third_party/move/tools/move-spec-test/src/report.rs

```diff
@@ -9,7 +9,7 @@ use tabled::{builder::Builder, settings::Style};
 #[derive(Debug, Serialize)]
 pub struct Report {
     /// The list of entries in the report.
-    files: HashMap<PathBuf, MutantStats>,
+    files: HashMap<PathBuf, Vec<MutantStats>>,
```

Comment:
The completely alternative approach (not sure if it is better or anything, just throwing possible inspirations here):
```suggestion
    files: HashMap<(PathBuf, MutantName), MutantStats>,
```

Reply:
Yeah, that's another possible grouping file with the modules.
I'll leave mine for now - as it's nicely looks inside the JSON.